roles/openshift_facts/tasks/main.yml

```diff
@@ -9,6 +9,10 @@
     l_is_atomic: "{{ ostree_booted.stat.exists }}"
 - set_fact:
     l_is_containerized: "{{ (l_is_atomic | bool) or (containerized | default(false) | bool) }}"
+    l_is_system_containers: "{{ (use_system_containers | default(None) | bool) }}"
```

Comment:
I thought that was a bit too "aggressive" and still have the possibility to install as Docker containers if needed.  Maybe we should keep both for a while?